src/test/java/uk/gov/hmcts/reform/draftstore/service/crypto/CryptoServiceTest.java

```diff
@@ -43,4 +43,30 @@ public void should_throw_an_exception_when_wrong_secret_is_used() throws Excepti
         assertThat(thrown)
             .isInstanceOf(InvalidKeyException.class);
     }
+
+    @Test
+    public void should_throw_an_exception_when_empty_input_received() {
```

Comment:
It would be ideal to have two different tests - one for encrypting and one for decrypting. Not strong about it, so just to consider.

Reply:
it would make sense if those tests actually are responsible for encrypting/decrypting. but the sole purpose is to cover the branch of conditional exception throw disregarding the following code coherence